tests/test_geolocation.py

```diff
@@ -178,7 +179,8 @@ def test_four_points_non_collinear(self):
 
     def test_more_than_four_points_one_correct_case(self):
         coordinatesArray = np.array([[0, 0], [10, 12], [-1, -1], [100, 0], [4, 4]])
-        expected = np.array([[0, 0], [10, 12], [-1, -1], [100, 0]])
+        # expected = np.array([[0, 0], [10, 12], [-1, -1], [100, 0]])
```

Comment:
Remove the line.

I also would like you to add one more case: `test_more_than_four_points_last_correct_case` with:
* `coordinatesArray = np.array([[0, 0], [1, 1], [100, 0], [-1, -1], [4, 4], [2, -2], [-4, 10]])`
* `expected = np.array([0, 1, 2, 6])`